src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+public class DukeException extends Exception{
+    protected String message;
+
+    /** Constructs a new duke exception with the specified detail message.
+     *
```

Comment:
Most likely an accident, the JavaDocs description should be 1 line after the initial /** .